go/pvl/interp_test.go

```diff
@@ -2096,7 +2096,7 @@ func runPvlTest(t *testing.T, unit *interpUnitTest) {
 			fail("proof should have failed")
 		}
 
-		err = nil
+		// err is definitely nil here.
```

Comment:
since this is a test I don't care. But this makes the code fragile since it's easy to unwittingly change this invariant by changing distant code above

Reply:
True, but in general if `err` is ever `nil` the function should be returning (or the test failiing) anyway, right?  So we wouldn't even get down here.

Reply:
Assuming you meant if `err` is ever not-nil. Sometimes errors are logged and discarded. If `err` is used then it can end up non-nil even though the function should keep chugging. Generally, not here.

Reply:
Yeah true, though I'd argue that `err` should be set to `nil` only in the place where it's explicitly being ignored, like:

```go
if err != nil {
   log.CDebugf(ctx, "This is the error: %+v", err)
  err = nil
}
```

That seems most foolproof and lintable.